BREAKING.md

```diff
@@ -25,6 +26,9 @@ The `OdspDocumentInfo` type is removed from `odsp-driver` package. It is removed
 ### Removed bindToContext from IFluidDataStoreChannel
 bindToContext in IFluidDataStoreChannel was deprecated in [0.50](#Deprecated-bindToContext-in-IFluidDataStoreChannel). This has now been removed.
 
+### Replace `createCreateNewRequest` with `createOdspCreateContainerRequest`
+The `createCreateNewRequest` function is removed from `odspDriverUrlResolver.ts` and `odspDriverUrlResolverForShareLink.ts` files under the `odsp-driver` package. `createCreateNewRequest` function is replaced with `createOdspCreateContainerRequest` function in `odspCreateContainer.spec.ts` and `odspDriverResolverTest.spec.ts`. The update is also made in `webpack-fluid-loader` package. If any instances of `createCreateNewRequest` function are used, replace them with `createOdspCreateContainerRequest` function by importing it from `@fluidframework/odsp-driver` package.
```

Comment:
Customers reading this don't know our file names - instead provide the package names and class/function names that they might be using.

No need to include test changes here, since our customers won't be referring to them.

Also no need to include changes to webpack-fluid-loader because it's under the `@fluid-tools` package scope, which does not publish publicly (so customers are not using it).